pennylane/transforms/qcut.py

```diff
@@ -853,3 +857,85 @@ def qcut_processing_fn(
         tensors, communication_graph, prepare_nodes, measure_nodes, use_opt_einsum
     )
     return result
+
+
+@batch_transform
+def cut_circuit(tape: QuantumTape, **kwargs) -> Tuple[Tuple[QuantumTape], Callable]:
+    """
+    Batch transform for circuit cutting.
+
+    .. note::
+
+        This function is designed for use as part of the circuit cutting workflow. Check out the
+        :doc:`transforms </code/qml_transforms>` page for more details.
+
+    Args:
+        tape (QuantumTape): The tape of the full circuit to be cut.
+
+    Returns:
+        the tapes corresponding to the circuit fragments as a result of cutting
+        and a post-processing function which combines the results via tensor contractions.
+
+        **Example**
+
+        Consider the following circuit containing a :class:`~.WireCut` operation:
+
+        .. code-block:: python
+
+            dev = qml.device("default.qubit", wires=2)
+
+            @qml.qnode(dev)
+            def circuit(x):
+                qml.RX(x, wires=0)
+                qml.RY(0.543, wires=1)
+                qml.WireCut(wires=0)
+                qml.CNOT(wires=[0, 1])
+                qml.RZ(0.240, wires=0)
+                qml.RZ(0.133, wires=1)
+                return qml.expval(qml.PauliZ(wires=[0]))
+
+            >>> x = 0.531
+            >>> print(circuit(x))
+            0.8623011058543121
+            >>> print(qml.grad(circuit)(x))
+            -0.506395895364911
+
+        This can be cut using the following transform
+
+        .. code-block:: python
+
+            >>> x = 0.531
+            >>> cut_circuit = qcut.cut_circuit(circuit)
+            >>> cut_circuit(x)
+            0.8623011058543121
```

Comment:
For console-style codeblocks, we don't need the `.. code-block:: python` decorator.
```suggestion
        >>> x = 0.531
        >>> cut_circuit = qcut.cut_circuit(circuit)
        >>> cut_circuit(x)
        0.8623011058543121
```